icons/scissors-line-dashed.svg

```diff
@@ -0,0 +1,19 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M5.4 9.4 8 12" />
+  <circle cx="4" cy="8" r="2" />
+  <path d="m14 6-8.6 8.6" />
+  <circle cx="4" cy="16" r="2" />
+  <path d="M10.8 14.8 14 18" />
+  <path d="M16 12h-2" />
+  <path d="M22 12h-2" />
+</svg>
```

Comment:
All of these could also do with an extra decimal of precision:
![image](https://github.com/lucide-icons/lucide/assets/17746067/33dfb823-3067-4543-9002-6fc83920b3a4)